Documentation/policy/language.rst

```diff
@@ -444,6 +444,12 @@ connection are either managed by Cilium or use an IP belonging to a node in the
 cluster (including host networking pods). This traffic may be allowed using
 labels, services or entities -based policies as described above.
 
+When CIDR policies are applied, the identity generated is local to each node.
+In other words, the identity may not be the same for the same CIDR policy
+across two nodes. Additionally, CIDR identities are not included within the
+numerical identities range (256 - 2^16 -1), as it is stored separately inside
+the Agent.
```

Comment:
I am not so sure about the position of this. The overall document is describing the policy language, and then how policies select different things and how we choose which peers to select. This paragraph brings in very specific implementation details around identities which seems a bit out-of-place to me.

Reply:
Yeah, I initially thought that too and hoped people would point it out. Since at least one other person thinks it's a bit out-of-place, I'll move it into the newly created internals section (second commit). :+1: